crates/mock-download-server/Cargo.toml

```diff
@@ -12,3 +12,10 @@ serde = { version = "1.0.218", features = ["derive"] }
 serde_json = "1.0.139"
 tiny_http = { version = "0.12.0", default-features = false, features = ["rustls"] }
 time = { version = "0.3.37", features = ["std", "serde", "serde-well-known"] }
+sha2 = "0.10.7"
```

Comment:
Move this to a workspace dep, please, the `criticalup-cli` and `criticaltrust` packages also uses it.